OpenGL/Constructs/VAO.cs

```diff
@@ -443,6 +443,11 @@ public void Init(IGenericVBO[] vbos)
         private DrawElementsType elementType;
         private bool allowIntAsElementType = true;
 
+        /// <summary>
+        /// The offset into the element array buffer that this VAO begins.
+        /// </summary>
+        public int Offset { get; set; }
```

Comment:
Some of my comments below assume that this value is generally zero.  Let me know if this is not the case.